config/locales/client.en.yml

```diff
@@ -2454,7 +2454,10 @@ en:
       tags: "Tags"
       choose_for_topic: "choose optional tags for this topic"
       delete_tag: "Delete Tag"
-      delete_confirm: "Are you sure you want to delete that tag?"
+      delete_confirm:
+        one: "Are you sure you want to delete this tag and remove it from 1 topic it is assigned to?"
+        other: "Are you sure you want to delete this tag and remove it from {{count}} topics it is assigned to?"
+        no_topics: "Are you sure you want to delete this tag?"
```

Comment:
Could you please move `no_topics` to `delete_confirm_no_topics`? It shouldn't be a sub-key of `delete_confirm`.
Our [tests are failing on Travis](https://travis-ci.org/discourse/discourse/jobs/284214941) because we check that only `zero` and `other` keys exist. Other than that it looks fine!